src/lib.rs

```diff
@@ -319,30 +321,24 @@ impl EguiContext {
     /// Can accept either a strong or a weak handle.
     ///
     /// You may want to pass a weak handle if you control removing texture assets in your
-    /// application manually and you don't want to bother with cleaning up textures in egui.
+    /// application manually and you don't want to bother with cleaning up textures in Egui.
     ///
-    /// You'll want to pass a strong handle if a texture is used only in egui and there's no
+    /// You'll want to pass a strong handle if a texture is used only in Egui and there's no
     /// handle copies stored anywhere else.
-    pub fn set_egui_texture(&mut self, id: u64, texture: Handle<Image>) {
-        log::debug!("Set egui texture: {:?}", texture);
-        self.egui_textures.insert(id, texture);
+    pub fn add_image(&mut self, image: Handle<Image>) -> u64 {
```

Comment:
Can't we return a `egui::TextureId` directly? What will the user do with a u64?
* This would communicate more clearly what the return value actually means
* The consumer needs to care less about Egui internals.
AFAIK the only reasonable thing to do with this u64 is to put it in a `TextureId::User`.